Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -175,8 +186,44 @@ class ARExample: UIViewController {
         
         // Hide directions view if we're calibrating.
         userDirectionsView.isHidden = startCalibrating
+        
+        // Disable changing scenes if we're calibrating.
+        sceneItem.isEnabled = !startCalibrating
     }
-
+    
+    /// Sets up the required functionality in order to display the scene and AR experience represented by `sceneInfo`.
+    ///
+    /// - Parameter sceneInfo: The sceneInfo used to set up the scene and AR experience.
+    fileprivate func selectSceneInfo(_ sceneInfo: SceneInfoType?) {
```

Comment:
This is not a big deal, but I would recommend making the parameter non-optional since the function does nothing when it is `nil`. Having an optional `SceneInfoType` should be the caller's problem, not the callee's problem.